packages/web-components/fast-foundation/src/anchor/anchor.ts

```diff
@@ -98,12 +98,17 @@ export class DelegatesARIALink extends ARIAGlobalStatesAndProperties {
     public ariaExpanded: "true" | "false" | undefined;
 }
 
+applyMixins(DelegatesARIALink, ARIAGlobalStatesAndProperties);
+
 /**
  * Mark internal because exporting class and interface of the same name
  * confuses API documenter.
  * TODO: https://github.com/microsoft/fast/issues/3317
  * @internal
  */
 /* eslint-disable-next-line */
-export interface Anchor extends StartEnd, DelegatesARIALink {}
+export interface Anchor
+    extends StartEnd,
+        DelegatesARIALink,
+        ARIAGlobalStatesAndProperties {}
 applyMixins(Anchor, StartEnd, DelegatesARIALink);
```

Comment:
Thanks @EisenbergEffect! Quick question since I'm just looking at the PR; does what's on 101 automatically get inherited on here on 114 with the name "DelegatesARIALink"?

Reply:
Yes. Line 101 mixes the globals into DelegatesARIALink so it can be used on its own as a complete set of attributes. Then line 114 mixes the entire thing into the Anchor.

Reply:
Great, thanks for the quick action on this!